### DISCUSSION TITLE
Thank you!!!

### DISCUSSION THREAD

**Discussion by toddrob99** [weltenwort/frigate-synology-dsm7#68] [General]
Thank you so much for the info and docker image you've published here. I have been going around in circles because the official Frigate image works fine on my DS1817+ but not the DS1821+ that I upgraded to. I must have removed and reinstalled the driver [100 times](https://github.com/robertklep/dsm7-usb-serial-drivers/issues/30). I even bought a pcie adapter and dual coral device to try that, but then I learned the apex driver is also not loaded in DSM7. While searching to see if anyone has built the apex driver for Synology, I came across this repo. I tried your image with the USB Coral and it's working! I sent a donation through GitHub Sponsors to show my appreciation.

**Comment by weltenwort** [comment]
I'm glad to hear that it solves a problem for you - thank you for the support! :heart:

**Comment by luigitare** [comment]
Thx a million for this!!!

**Comment by imne** [comment]
Thanks alot for this project been running ghcr.io/weltenwort/frigate-synology-dsm7:v0.11.0-rc1 for a few months on my DS920+ it's been working great but I see v0.11.1, gonna upgrade soon.

**Comment by jmtatsch** [comment]
@toddrob99 if you still have that pci-e coral around, could you try installing gasket directly on the Synology via nixos and this pr https://github.com/NixOS/nixpkgs/pull/197610 ?
I also have a DS920+ and am still deciding whether to buy the nice m2 version or the yucky usb version.

**Reply by toddrob99** [reply]
> @toddrob99 if you still have that pci-e coral around, could you try installing gasket directly on the Synology via nixos and this pr https://github.com/NixOS/nixpkgs/pull/197610 ?
> 
> I also have a DS920+ and am still deciding whether to buy the nice m2 version or the yucky usb version. 

Sorry, I sold it.
